.eslintrc.json

```diff
@@ -19,6 +19,7 @@
     "no-shadow-restricted-names": 2,
     "no-undef-init": 2,
     "no-undefined": 2,
+    "no-use-before-define": 2,
```

Comment:
Honestly I don’t remember I just copied from Skaled, do you think we don’t need this one?.

> On Apr 25, 2016, at 5:58 PM, Adam Fenton <notifications@github.com> wrote:
> 
> angular